automation/test.sh

```diff
@@ -371,7 +373,6 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ sig-compute-migrations ]]; then
     export KUBEVIRT_E2E_FOCUS="Migration"
     export KUBEVIRT_E2E_SKIP="GPU|MediatedDevices"
-    export KUBEVIRT_STORAGE="rook-ceph"
```

Comment:
@rmohr FYI

Reply:
Yeah @dhiller @xpivarc I was wrong, this has implications

Reply:
I apologize for my blindness. Can you point to the exact place?

Reply:
https://github.com/kubevirt/kubevirt/blob/5a734f92980248744ef02949a875cfe02c1c1682/automation/test.sh#L394-L399

Reply:
Now, instead of overriding the focus var, I am appending storage-req to it. WDYT? @dhiller @xpivarc

Reply:
`if [[ ""rook-ceph"" == "rook-ceph-default" ]]; then` ?